server/tests/trim_gap_handling.rs

```diff
@@ -47,7 +49,11 @@ async fn fast_forward_over_trim_gap() -> googletest::Result<()> {
     base_config.bifrost.default_provider = Replicated;
     base_config.common.log_filter = "restate=debug,warn".to_owned();
     base_config.common.log_format = LogFormat::Compact;
-
+    // since default partition replication is not "everywhere" anymore,
+    // we need to explicitly set this to 3
+    base_config.admin.default_partition_replication = Some(ReplicationProperty::new(
```

Comment:
There is noway to describe "everywhere" anymore via the config, Even if the system still fully supports the "Everywhere" policy for backward compatibility.

When migrating from 1.2.2 to 1.3 (if this PR made it to 1.3) everything will continue to work with "everywhere" policy. Once set to something else it's not possible to go back to "everywhere"